CONTRIBUTING.md

```diff
@@ -3,20 +3,46 @@
 We're always looking for your help to fix bugs and improve the product. Create a pull request and we'll be happy to take a look.
 Start by reading the [Engineering Design](docs/HighLevelDesign.md). You can find the doxygen generated documentation [here](https://microsoft.github.io/onnxruntime/).
 
-# Checkin procedure
-1. Fork the repo
-2. git clone your fork
-3. Create feature branch
-4. Make and checkin your changes along with unit tests
-5. git commit your changes
-6. git push origin HEAD
-7. To request merge into master send a pull request from the web ui
-https://github.com/Microsoft/onnxruntime.
-8. Add 'Microsoft/onnxruntime' as a reviewer.
-
-New code *must* be accompanied by unit tests.
-
-*Note*: After creating a pull request, you might not see a build getting triggered right away. One of the
+# Proposing new public APIs
```

Comment:
Formatting - can you make all the headers except the top level Contributing level 2 (two #)?

Reply:
done

---
In reply to: [390701908](https://github.com/microsoft/onnxruntime/pull/3149#discussion_r390701908) [](ancestors = 390701908)